test/unit/view/connected-droppable.spec.js

```diff
@@ -60,9 +61,14 @@ type DragArgs = {|
   isDraggingOver: false | 'home' | 'foreign'
 |}
 
+const margin: Spacing = {
```

Comment:
the tests where asserting the correct values - but because there was no margin here `withMargin` was the same as `withoutMargin`